device/iothub_client_python/src/iothub_client_python.cpp

```diff
@@ -53,6 +53,7 @@
 #endif
 
 #define VERSION_STRING "1.4.5"
+#define PYTHON_PRODUCT_INFO "python/" VERSION_STRING
```

Comment:
>VERSION_STRING [](start = 38, length = 14)

Is there any other info that would be useful to get here?  The C SDK gives you a bunch of stuff free (see e.g. [https://github.com/Azure/azure-c-shared-utility/blob/1d622902d7842f94193fc394987f2b4e978bb700/adapters/platform_win32.c#L62](https://github.com/Azure/azure-c-shared-utility/blob/1d622902d7842f94193fc394987f2b4e978bb700/adapters/platform_win32.c#L62) for Win32).  One obvious would be which version of Python you're running (2 vs 3) and maybe there's other no-PII Python specific settings that may be useful??